lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -116,9 +116,10 @@
                                 title: gettext('Time Zone'),
                                 valueAttribute: 'time_zone',
                                 helpMessage: gettext(
-                                    'Select the time zone for displaying course dates. If you do not specify a ' +
-                                    'time zone here, course dates, including assignment deadlines, are displayed in ' +
-                                    'Coordinated Universal Time (UTC).'
+                                    'Select the time zone for displaying course dates. ' +
+                                    'If you do not specify a time zone, course dates, ' +
+                                    'including assignment deadlines, will be displayed ' +
+                                    'in your browser\'s local time zone.'
```

Comment:
Actually nvm, this isn't a StringUtils fix, it's a "make this line extra long and disable the quality failure for that because gettext is stupid" fix.

See https://github.com/edx/edx-platform/search?&q=max-len for many other examples.